lib/internal/per_context/primordials.js

```diff
@@ -19,13 +30,17 @@ const { bind, call } = Function.prototype;
 const uncurryThis = call.bind(bind, call);
 primordials.uncurryThis = uncurryThis;
 
+// Because some of these functions are used by `makeSafe`, which is exposed
+// on the `primordials` object, it's important to use const references
+// to the primordials that they use:
+/* eslint-enable node-core/prefer-primordials */
```

Comment:
nit: I think this line should go above the previous comment, just under line 31.